packages/editor-ui/src/components/V1Banner.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+	<n8n-callout
+		v-if="shouldDisplay"
+		theme="warning"
+		icon="info-circle"
+		override-icon
+		:class="$style['v1-banner']"
+	>
+		<span v-html="locale.baseText('banners.v1.message')"></span>
+		{{ '' }}
+		<a v-if="isInstanceOwner" @click="dismissBanner('v1', 'permanent')">
+			<span v-html="locale.baseText('banners.v1.action')"></span>
+		</a>
+		<template #trailingContent>
+			<n8n-icon
+				size="small"
+				icon="xmark"
+				:title="locale.baseText('banners.v1.iconTitle')"
+				:class="$style.xmark"
+				@click="dismissBanner('v1', 'temporary')"
+			/>
+		</template>
+	</n8n-callout>
+</template>
+
+<script setup lang="ts">
+import { VIEWS } from '@/constants';
+import { computed } from 'vue';
+import { useUIStore, useUsersStore, useRootStore } from '@/stores';
+import { useRoute } from 'vue-router/composables';
+import { i18n as locale } from '@/plugins/i18n';
+
+const { isInstanceOwner } = useUsersStore();
+const { dismissBanner } = useUIStore();
+
+const shouldDisplay = computed(() => {
+	if (!useRootStore().versionCli.startsWith('1.')) return false;
+
+	if (useUIStore().banners.v1.dismissed) return false;
+
+	const VIEWABLE_AT: string[] = [
+		VIEWS.HOMEPAGE,
+		VIEWS.COLLECTION,
+		VIEWS.TEMPLATE,
+		VIEWS.TEMPLATES,
+		VIEWS.CREDENTIALS,
+		VIEWS.VARIABLES,
+		VIEWS.WORKFLOWS,
+		VIEWS.EXECUTIONS,
+	];
+
+	const { name } = useRoute();
+
+	if (name && VIEWABLE_AT.includes(name)) return true;
+
+	return false;
+});
+</script>
+
+<style module lang="scss">
+.v1-banner {
+	position: absolute;
```

Comment:
we could use `position: sticky` instead, or even better if we can fit this banner in the existing grid defined in `App.vue`.

Reply:
With `absolute` the banner already [remains in viewport on scroll](https://share.cleanshot.com/pWWg4N4x), which I think is what we want. See e.g. `/templates` 

With `sticky` the banner does not render - I'd need to look into the stacking context to debug, plus I'd rather keep it out of the flow of the document to minimize the chance of causing layout issues. So for MVP I think it's okay? Let me know otherwise.

Reply:
with `sticky` to work, you'd need to move `<V1Banner />` to line 3 in `App.vue`.

Reply:
That [blocks out scrolling](https://share.cleanshot.com/YsLLCrlB) in the templates feed.